lang/python/python-pycparser/patches/001-use-external-ply.patch

```diff
@@ -13,19 +13,19 @@
  class CLexer(object):
 --- a/pycparser/c_parser.py
 +++ b/pycparser/c_parser.py
-@@ -8,7 +8,7 @@
+@@ -6,7 +6,7 @@
+ # Eli Bendersky [https://eli.thegreenplace.net/]
+ # License: BSD
```

Comment:
i'd try to send this upstream eventually;
it's been in our tree for 4-5 years now;